src/components/headers/collection-header.tsx

```diff
@@ -676,6 +760,39 @@ export class CollectionHeader extends React.Component<IProps, IState> {
       });
   };
 
+  private deprecate(collection) {
+    const operation = !collection.deprecated;
+    const operation_text = operation ? t`deprecated` : t`undeprecated`;
+
+    CollectionAPI.setDeprecation(
+      collection,
+      operation,
+      this.context.selectedRepo,
+    )
+      .then((res) => {
+        const taskId = getIdFromTask(res.data.task);
+        waitForTask(taskId).then(() => {
+          if (this.props.reload) {
+            this.props.reload();
+          }
+        });
+      })
+      .catch((err) => {
+        const { status, statusText } = err.response;
+        this.setState({
+          collectionVersion: null,
+          alerts: [
+            ...this.state.alerts,
+            {
+              variant: 'danger',
+              title: t`Collection "${collection.name}" could not be "${operation_text}".`,
```

Comment:
```suggestion
              title: !collection.deprecated ? t`Collection "${collection.name}" could not be deprecated.` : t`Collection "${collection.name}" could not be undeprecated.`,
```

computing the verb of the sentence could be problematic for translations